cli/src/test/resources/fixture/impacted_targets_distances-1-2.txt

```diff
@@ -0,0 +1,8 @@
+//test/java/com/integration:libbazel-diff-integration-test-lib-src.jar~0~0
+//test/java/com/integration:TestStringGenerator.java~0~0
+//test/java/com/integration:bazel-diff-integration-test-lib~0~0
+//test/java/com/integration:bazel-diff-integration-tests-src.jar~2~0
+//test/java/com/integration:libbazel-diff-integration-test-lib.jar~0~0
+//test/java/com/integration:bazel-diff-integration-tests~1~0
+//test/java/com/integration:bazel-diff-integration-tests_deploy-src.jar~2~0
+//test/java/com/integration:bazel-diff-integration-tests.jar~2~0
```

Comment:
I just copied one of the other testcases & expected output, then updated this file as needed to get the test to pass.

I'd like to have a better e2e test that shows more interesting distance measurements results. How did you generate these testcase workspaces?

Reply:
I don't have any context on this, @bz-canva has the most knowledge

Reply:
I ended up removing this test file and testcase that used it in favor of vendoring a new test workspace and running on that.